src/containers/Home/styles.js

```diff
@@ -321,7 +317,7 @@ export const ServicesWrapper = styled.div`
   }
 `;
 export const Banner = styled.div`
-  padding-bottom: 110px;
+padding-bottom: 110px;
```

Comment:
use theme spacing thought out all the pages

Reply:
Used theme spacing all over.